specification/batch/data-plane/Microsoft.Batch/2017-09-01.6.0/BatchService.json

```diff
@@ -11172,6 +11195,11 @@
           "title": "The command line of the Job Manager task.",
           "description": "The command line does not run under a shell, and therefore cannot take advantage of shell features such as environment variable expansion. If you want to take advantage of such features, you should invoke the shell in the command line, for example using \"cmd /c MyCommand\" in Windows or \"/bin/sh -c MyCommand\" in Linux."
         },
+        "containerSettings": {
+          "$ref": "#/definitions/TaskContainerSettings",
+          "title": "The settings for the container under which the Job Manager task runs.",
+          "description": "If the pool that will run this task has containerConfiguration set, this is required. When this is specified, all directories recursively below the AZ_BATCH_NODE_ROOT_DIR (the root of batch directories on the node) are mapped into the container, all task enviornment variables are mapped into the container, and the task command line is executed in the container."
```

Comment:
If the pool _doesn't_ have containerConfiguration then this must _not_ be present?  Or can containers be introduced into non-container pools?

Reply:
No, it's basically "must match"

I'll reword to clarify.